src/sections/aliases.tsx

```diff
@@ -0,0 +1,211 @@
+import { Paper, Table, TableBody, TableCell, TableContainer, TableHead, TableRow } from "@mui/material";
+import Section from "../future-hopr-lib-components/Section";
+import { useAppDispatch, useAppSelector } from "../store";
+import { useEffect, useRef, useState } from "react";
+import { actionsAsync } from "../store/slices/node/actionsAsync";
+import { exportToCsv } from "../utils/helpers";
+
+function AliasesPage() {
+  const dispatch = useAppDispatch();
+  const aliases = useAppSelector(selector => selector.sdk.aliases)
+  const loginData = useAppSelector(selector => selector.auth.loginData)
+
+  useEffect(() => {
+    if (loginData.ip && loginData.apiKey) {
+      dispatch(actionsAsync.getAliasesThunk({ apiEndpoint: loginData.ip, apiToken: loginData.apiKey }))
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [loginData])
+
+  return (
+    <Section
+      className="Section--aliases"
+      id="Section--aliases"
+      yellow
+    >
+      <h2>Add new alias</h2>
+      <CreateAliasForm />
+      <h2>Aliases table</h2>
+      <button onClick={() => {
+        if (aliases) {
+          exportToCsv(Object.keys(aliases).map(alias => ({
+            alias: alias,
+            peerId: aliases[alias]
+          })), "aliases.csv")
+        }
+      }}>export</button>
+      <CSVUploader onParse={(parsedData) => {
+        for (const data of parsedData) {
+          if (data.alias && data.peerId && loginData.ip && loginData.apiKey) {
+            dispatch(actionsAsync.setAliasThunk({
+              alias: String(data.alias),
+              peerId: String(data.peerId),
+              apiEndpoint: loginData.ip,
+              apiToken: loginData.apiKey
+            }))
+          }
+        }
+      }} />
+      <TableContainer component={Paper}>
+        <Table sx={{ minWidth: 650 }} aria-label="aliases table">
+          <TableHead>
+            <TableRow>
+              <TableCell>id</TableCell>
+              <TableCell>peerId</TableCell>
+              <TableCell>alias</TableCell>
+              <TableCell>actions</TableCell>
+            </TableRow>
+          </TableHead>
+          <TableBody>
+            {Object.entries(aliases ?? {}).map(([alias, peerId], key) => (
+              <TableRow
+                key={key}
+              >
+                <TableCell component="th" scope="row">
+                  {key}
+                </TableCell>
+                <TableCell>{peerId}</TableCell>
+                <TableCell>{alias}</TableCell>
+                <TableCell><DeleteAliasButton alias={alias} /></TableCell>
```

Comment:
We need a state in the component in which we will store 'table action errors'.
If this call fails, save the message to this state.
In next PRs we will add Notification center or information snackbar center.